library/src/scripts/icons/icons.story.tsx

```diff
@@ -320,6 +320,21 @@ story.add("Icons", () => {
                     <RevisionStatusPublishedIcon />
                 </StoryTileAndTextCompact>
             </StoryTiles>
+            <StoryHeading>New Post Menu</StoryHeading>
+            <StoryTiles>
+                <StoryTileAndTextCompact text={`NewPostMenuIcon`}>
+                    <CommonIcons.NewPostMenuIcon />
```

Comment:
Since this is white on white, put a light grey background color to the container. I think we have a few examples